doc/techref/projections.md

```diff
@@ -0,0 +1,50 @@
+---
+myst:
+  substitutions:
+    lon0: "lon{sub}`0`",
+    lat0: "lat{sub}`0`",
+    lon1: "lon{sub}`1`",
+    lat1: "lat{sub}`1`",
+    lat2: "lat{sub}`2`",
+    lonp: "lon{sub}`p`",
+    latp: "lat{sub}`p`",
+    url_doc_proj: "https://www.pygmt.org/latest/projections/",
```

Comment:
Remove the trailing comma:
```suggestion
    lon0: "lon{sub}`0`"
    lat0: "lat{sub}`0`"
    lon1: "lon{sub}`1`"
    lat1: "lat{sub}`1`"
    lat2: "lat{sub}`2`"
    lonp: "lon{sub}`p`"
    latp: "lat{sub}`p`"
    url_doc_proj: "https://www.pygmt.org/latest/projections/"
```

Reply:
Oh no, you are right! I did not spot this difference to the definition within the `conf.py` file yesterday, despite checking this several times :see_no_evil: :see_no_evil:.